### DISCUSSION TITLE
Seq stopped responding with A request was aborted

### DISCUSSION THREAD

**Discussion by cainef** [datalust/seq-tickets#1939] [Help]
Hi,
We have recently migrated from Seq on Windows to docker on Linux.
This morning users reported that Seq was unavailable and on checking the docker logs I am seeing 'A request was aborted' constantly repeated in the logs.

I'm not sure how to turn on debug log level to get more info.

I have check the server and all basic resource usage (cpu, memory, disk) looks OK.

Can you advise what else I should be looking into here?

Thanks

**Comment by nblumhardt** [comment]
Howdy! If the UI is reachable, then the diagnostic report under Settings > Diagnostics > Download a diagnostic report will have just about everything that's collected on the Seq side.

The other two places to investigate are Data > Ingestion, and Data > Storage.

Outside of Seq, you might need to investigate ingress controllers, proxies/app gateway, and any other network infrastructure between your users and Seq, but nothing specific to Seq at that level.

If none of those appear to contain any clues, mailing support@datalust.co would be the next step.

Hope this helps!
Nick
